From 957d7e40526636b06e973085db70a9941a6bdede Mon Sep 17 00:00:00 2001
From: Martin Abente Lahaye <tch@sugarlabs.org>
Date: Tue, 13 May 2014 11:13:13 -0400
Subject: [PATCH 2/2] Fix KeyValuesDialog password visibility

Check whether or not the key has a keyword 'password' in it,
and then set visibility to False.

Signed-off-by: Martin Abente Lahaye <tch@sugarlabs.org>
---
 src/jarabe/desktop/keydialog.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/jarabe/desktop/keydialog.py b/src/jarabe/desktop/keydialog.py
index f8ecf5a..54450fa 100644
--- a/src/jarabe/desktop/keydialog.py
+++ b/src/jarabe/desktop/keydialog.py
@@ -97,6 +97,8 @@ class NetworkParameters(Gtk.HBox):
 
         if self._is_entry():
             self._entry = Gtk.Entry()
+            if 'password' in self._key:
+                self._entry.set_visibility(False)
             self.pack_start(self._entry, True, True, 0)
             self._entry.show()
         elif self._is_liststore():
-- 
1.8.3.1

